save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/NameFiltersRow.kt

```diff
@@ -0,0 +1,87 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.basic.table.filters
+
+import com.saveourtool.save.frontend.externals.fontawesome.*
+import react.FC
+import react.Props
+import react.dom.html.ReactHTML.button
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.input
+import react.useEffect
+import react.useState
+import web.cssom.ClassName
+import web.html.ButtonType
+import web.html.InputType
+
+val nameFiltersRow: FC<NameFilterRowProps> = FC { props ->
+    val (filtersName, setFiltersName) = useState(props.name)
+    useEffect(props.name) {
+        if (filtersName != props.name) {
+            setFiltersName(props.name)
+        }
+    }
+
+    div {
+        className = ClassName("container-fluid")
+        div {
+            className = ClassName("row d-flex")
+
+            div {
+                className = ClassName("col-0 mr-3 align-self-center")
+                fontAwesomeIcon(icon = faFilter)
+            }
+            div {
+                className = ClassName("row")
+                div {
+                    className = ClassName("col-auto align-self-center")
+                    +"Name: "
+                }
+                div {
+                    className = ClassName("col-auto mr-3")
+                    input {
+                        type = InputType.text
+                        className = ClassName("form-control")
+                        value = filtersName ?: ""
+                        required = false
+                        onChange = {
+                            setFiltersName(it.target.value)
+                        }
+                    }
+                }
+            }
+            button {
+                type = ButtonType.button
+                className = ClassName("btn btn-secondary mr-3")
+                fontAwesomeIcon(icon = faSearch)
+                onClick = {
+                    props.onChangeFilters(filtersName)
+                }
+            }
```

Comment:
```suggestion
            buttonBuilder(faSearch, style = "secondary", classes = "mr-3") {
                props.onChangeFilters(filtersName)
            }
```

Reply:
If you can see - I have updated new filters in new code to button builder, this is old one...
I will upgrade ALL buttons to buttonBuilder a little bit later in one PR

Reply:
oh ok got it